pkg/authorize/tollbooth/mock.go

```diff
@@ -81,3 +83,9 @@ func Write(w http.ResponseWriter, statusCode int, resp interface{}) error {
 	w.Write(data)
 	return nil
 }
+
+func fnvHash(text string) string {
```

Comment:
we have two identical implementations of this, one in authorize/stub and another in authorize/tollbooth.mock. Maybe we should mark one as exported and share it.

Reply:
addressed in https://github.com/openshift/telemeter/pull/56